### ISSUE TITLE
Possible to add limits to sliders?

### ISSUE THREAD

**Issue by TheBasedDoge** [mattieha/slider-button-card#126]
**Is your feature request related to a problem? Please describe.**

Possible to set limits for sliders, min and max. The issue is my slider is actually fully open at 75%, 100% is overextended.

**Describe the solution you'd like**

Ability to set a limit on the slider, so for example 100% on the slider is actually 75% for the device.

maybe this is already implemented but I cannot figure out how to do it.

**Comment by guzzard** [comment]
Yes, would be really nice to be able to set minimum 1% on lamps as well, so you can just swipe fast without turning off the light completely.

**Comment by TheBasedDoge** [comment]
Yeah I still have found no solution for this, tried several slider plugins with no luck. :(

**Comment by guzzard** [comment]
@TheBasedDoge https://github.com/thomasloven/lovelace-slider-entity-row has min/max

**Comment by TheBasedDoge** [comment]
> @TheBasedDoge https://github.com/thomasloven/lovelace-slider-entity-row has min/max

That worked!! thanks. I wish it had a vertical slider, but I'll take what I can get at this point.
